### DISCUSSION TITLE
3.0.0-SNAPSHOT

### DISCUSSION THREAD

**Discussion by nikosgram** [nikosgram/gringotts#184] [Announcements]
## What's Changed
* chore: create CODE_OF_CONDUCT.md by @nikosgram in https://github.com/nikosgram/gringotts/pull/158
* [Pull Request] Small addon to the gringott's config by @iomatix in https://github.com/nikosgram/gringotts/pull/168
* Fix another NPE with Shulker boxes and a fix for vault sign validation by @weaves7 in https://github.com/nikosgram/gringotts/pull/173
* AccountChest logic changes by @GroobleDierne in https://github.com/nikosgram/gringotts/pull/174
* fix MC 1.21 plugin load failure - issue #176 by @DocSplinters in https://github.com/nikosgram/gringotts/pull/178
* permissions chapter: put lines of text in consistent order by @DocSplinters in https://github.com/nikosgram/gringotts/pull/179

## New Contributors
* @iomatix made their first contribution in https://github.com/nikosgram/gringotts/pull/168
* @weaves7 made their first contribution in https://github.com/nikosgram/gringotts/pull/173
* @DocSplinters made their first contribution in https://github.com/nikosgram/gringotts/pull/178

**Full Changelog**: https://github.com/nikosgram/gringotts/compare/2.12.6-SNAPSHOT...3.0.0-SNAPSHOT

<hr /><em>This discussion was created from the release <a href='https://github.com/nikosgram/gringotts/releases/tag/3.0.0-SNAPSHOT'>3.0.0-SNAPSHOT</a>.</em>

**Comment by Relaxing9** [comment]
Is this backwards compatible with versions such as Minecraft 1.20.4?

Edit: need it for 1.20.6 now...

**Reply by GroobleDierne** [reply]
No but I back ported it here https://github.com/LaBoulangerie/Server/blob/main/plugins%2Fgringotts.jar
This version also contains my last PR that isn't merged yet with important bug fixes and performance improvements.

**Reply by Relaxing9** [reply]
> No but I back ported it here https://github.com/LaBoulangerie/Server/blob/main/plugins%2Fgringotts.jar This version also contains my last PR that isn't merged yet with important bug fixes and performance improvements.

That link doesn't work for me. I would really like to download the latest version because the current one has quite some issues for me: https://paste.gg/p/anonymous/1d8f5d8a94e2438eaf8f3ff6a61a7ef9
